bcl/src/bc_module_battery.c

```diff
@@ -59,10 +59,12 @@ void bc_module_battery_init(bc_module_battery_format_t format)
     }
 
     bc_gpio_init(BC_GPIO_P1);
-    _bc_module_battery_measurement(DISABLE);
+
     bc_gpio_set_mode(BC_GPIO_P1, BC_GPIO_MODE_OUTPUT);
+    bc_gpio_set_output(BC_GPIO_P1, 0);
```

Comment:
Please put `bc_gpio_set_output` before `bc_gpio_set_mode`. The rest is fine, feel free to merge it yourself after this change. Thank you.